sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/StoreResult.java

```diff
@@ -159,7 +160,9 @@ public String toString() {
                 ", sessionToken: " + (this.sessionToken != null ? this.sessionToken.convertToString() : null) +
                 ", exception: " + BridgeInternal.getInnerErrorMessage(this.exception);
     }
-    public static class StoreResultSerializer extends StdSerializer<StoreResult> {
+    public static class StoreResultSerializer extends StdSerializer<StoreResult> implements Serializable {
```

Comment:
StdDeserializer already implements Serializable

Reply:
that's right. But spotbug still want to inherited types to explicitly say it, not sure why so.

Reply:
There are other places i saw StdDeserializer without extra implements , so i think only serializerId is needed and this can be remove , please give it a try

Reply:
@simplynaveen20 thank you, I double-checked, yes, only the serializationVersionId is required. I've updated the PR that removes extra implements.